common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -367,7 +367,22 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
   }
 
   def dynamicConfigStoreBackend: String = get(DYNAMIC_CONFIG_STORE_BACKEND)
-  def dynamicConfigRefreshInterval: Long = get(DYNAMIC_CONFIG_REFRESH_INTERVAL)
+  def dynamicConfigRefreshInterval: Long = get(DYNAMIC_CONFIG_REFRESH_INTERVAL).getOrElse(-1L)
```

Comment:
why not just return a `Option[Long]`?

Reply:
> why not just return a `Option[Long]`?

For java code we will get Option[Object] which may cause little boring.

Reply:
Emm... alright